resources/views/components/menu/management.blade.php

```diff
@@ -1,83 +1,87 @@
-            <div class="dropdown-divider"></div>
-        @endcan
-        {{--@if(request()->user()->role_id >= \App\Site\Models\Role::Admin)
-            <h6 class="dropdown-header">Administration</h6>
-            <a class="dropdown-item" href="{{ route('admin.dashboard') }}">Admin</a>
-            <div class="dropdown-divider"></div>
+        @can('develop')
+            @can('manage', App\Community\Models\TriggerTicket::class)
+                <x-dropdown-header>{{ __('Development') }}</x-dropdown-header>
+                {{--<x-dropdown-item :link="route('triggers.ticket.index')">{{ __res('ticket') }}</x-dropdown-item>--}}
+                <x-dropdown-item :link="url('ticketmanager.php')">{{ __res('ticket') }}</x-dropdown-item>
+                <x-dropdown-item :link="url('ticketmanager.php?f=1')">Most Reported Games</x-dropdown-item>
+                <x-dropdown-item :link="url('achievementinspector.php')">Achievement Inspector</x-dropdown-item>
+            @endcan
+            @can('manage', App\Community\Models\AchievementSetClaim::class)
+                <x-dropdown-item :link="url('expiringclaims.php')">Expiring Claims</x-dropdown-item>
+            @endcan
+            @can('manage', App\Platform\Models\GameHash::class)
+                <x-dropdown-item :link="url('latesthasheslinked.php')">Latest Linked Hashes</x-dropdown-item>
+            @endcan
         @endif
-        @can('manage', App\Site\Models\Event::class)
-            <h6 class="dropdown-header">Events</h6>
-            <div class="dropdown-divider"></div>
-        @endcan--}}
-        @can('manage', App\Platform\Models\IntegrationRelease::class)
-            <h6 class="dropdown-header">Releases</h6>
-            @can('manage', App\Platform\Models\Emulator::class)
-                <a class="dropdown-item {{ request()->routeIs('emulator*') ? 'active' : '' }}"
-                   href="{{ route('emulator.index') }}">
-                    Emulators
-                </a>
+        @if($user->Permissions >= Permissions::Developer)
+            <x-dropdown-header>{{ __('Community') }}</x-dropdown-header>
+            @can('manage', App\Community\Models\News::class)
+                {{--<x-dropdown-item :link="route('news.index')">{{ __res('news') }}</x-dropdown-item>--}}
+                <x-dropdown-item :link="url('submitnews.php')">{{ __res('news') }}</x-dropdown-item>
             @endcan
-            <a class="dropdown-item {{ request()->routeIs('integration.release*') ? 'active' : '' }}"
-               href="{{ route('integration.release.index') }}">
-                Integration
-            </a>
-            <div class="dropdown-divider"></div>
-        @endcan
+            @can('manage', App\Community\Models\Forum::class)
+                {{--<x-dropdown-item :link="route('forum-topic.clearing')">Forum Clearing</x-dropdown-item>--}}
+                <x-dropdown-item :link="url('viewforum.php?f=0')">Forum Clearing</x-dropdown-item>
```

Comment:
While "Invalid Forum Posts" implies there's something wrong with them, this makes no sense to me. Maybe "Unvalidated Forum Posts" or "Unauthorised Forum Posts"?

Reply:
Ah, yes, that's a leftover term from the prototype. What do you think about "Forum Verification" to make "Forum" the first word?

Reply:
I'm fine with "Forum Verification".

Reply:
Maybe both?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/cd10eb86-e8cb-46d2-8e35-76e740aa5b44)

Reply:
Looks very good to me, let's go with that